lib/gateways.js

```diff
@@ -8,6 +8,7 @@ var cm = require('sdk/context-menu');
 var prefs = require('sdk/simple-prefs').prefs;
 var ioservice = Cc['@mozilla.org/network/io-service;1'].getService(Ci.nsIIOService);
 var gui = require('./gui.js');
+var tabs = require("sdk/tabs");
```

Comment:
We probably want single quotes here:
https://travis-ci.org/lidel/ipfs-firefox-addon/jobs/96923547#L790 
:)